docs/src/tutorials/bifurcation_diagram.md

```diff
@@ -47,11 +47,11 @@ bopts = ContinuationPar( dsmax = 0.05,        # Maximum arclength value of the p
                         detectBifurcation=3) # Value in {0,1,2,3} determining to what extent bifurcation points are detected (0 means nothing is done, 3 both them and there localisation are detected).
 ```
 With all this done, we can compute the bifurcation diagram:
-```julia
+```@example ex1
 bf = bifurcationdiagram(bprob, PALC(), 2, (args...) -> bopts)
 ```
 Finally, we can plot it:
-```julia
+```@example ex1
 plot(bf)
 ```
 ![bifurcation_diagram1](../assets/bifurcation_diagram.svg)
```

Comment:
If the example works then it should auto-generate the figure so this wouldn't be needed anymore.